app/src/main/java/com/google/droidjump/AchievementsFragment.java

```diff
@@ -65,18 +71,23 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         View rootView = getView();
         RecyclerView achievementsView = rootView.findViewById(achievementsViewId);
         LoadingHelper.onLoading(activity, rootView, achievementsViewId);
-        activity.getAchievementsClient().load(true).addOnCompleteListener(activity, task -> {
+        activity.getAchievementsClient().load(/* forceReload= */false).addOnCompleteListener(activity, task -> {
```

Comment:
Add space between comment and value 
/* forceReload= */false -> /* forceReload= */ false